test/cmdlineTests.sh

```diff
@@ -375,8 +375,13 @@ function test_via_ir_equivalence()
 
 ## RUN
 
-echo "Checking that the bug list is up to date..."
-"$REPO_ROOT"/scripts/update_bugs_by_version.py
+SOLTMPDIR=$(mktemp -d)
+printTask "Checking that the bug list is up to date..."
+cp "${REPO_ROOT}/docs/bugs_by_version.json" "${SOLTMPDIR}/original_bugs_by_version.json"
+"${REPO_ROOT}/scripts/update_bugs_by_version.py"
+diff --unified "${SOLTMPDIR}/original_bugs_by_version.json" "${REPO_ROOT}/docs/bugs_by_version.json" || \
+    fail "The bug list in bugs_by_version.json was out of date and has been updated. Please investigate and submit a bugfix if necessary."
```

Comment:
Shouldn't it be: `A bug list in bugs_by_version.json was out of date and has been updated...`?

Reply:
Depends. It would be `a` if you mean the list for a specific version but I by "bug list" I really mean the whole file.